### PR TITLE
Avoid raising raw 'Exception'

### PR DESCRIPTION
**Pull request by felixfontein**
Ref: https://github.com/ansible-community/antsibull-changelog/pull/98#discussion_r1096144470

### CODE REVIEW

File: src/antsibull_changelog/plugins.py [hunk 1]
```diff
@@ -433,8 +433,7 @@ def _get_ansible_core_version(paths: PathsConfig) -> packaging.version.Version:
         match = re.match(regex, output)
         if match:
             return packaging.version.Version(match.group(1))
-    # pylint:disable-next=broad-exception-raised
-    raise Exception(
+    raise ValueError(
```
Comment (gotmax23) [inline src/antsibull_changelog/plugins.py:436]: This feels more like a RuntimeError to me. [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError) is used for invalid input provided by a {C,A}PI user.

Comment (felixfontein) [reply src/antsibull_changelog/plugins.py]: Good idea! Adjusted in 74727958c25b79159f6c404e442a345d492e9f2e.

**Review by gotmax23 — APPROVED** [review]
Ack.

**Comment by gotmax23** [comment]
Thanks for this change! It makes my inner pythonista happy.

**Comment by felixfontein** [comment]
@gotmax23 thanks for reviewing and merging :)

**Pull request merged**